modules/stochastic_tools/test/tests/multiapps/batch_commandline_control/gold/master_multiple_out_storage_0002.csv

```diff
@@ -1,4 +1,4 @@
 sample_0
```

Comment:
Why are these numbers changing?

Reply:
I printed out the random number using the old code. I found the random number will change from first time to second time when a rand() is called. It will keep the same from second time. After I made the change, I think the random number remains the same every time rand() is called.